app/views/pages/api-reference/graphql/glossary.liquid

```diff
@@ -118,6 +118,8 @@ mutation edit_car($car_id: ID!, $color: String!) {
 }
 ```
 
+{% include 'alert/tip', content: 'Admin mutations and queries are used for interacting with instance code. They are an alternative for modifying partials, GraphQL, and others instead of editing files and deploying them. They are useful when you want to build more dynamic behavior.' %}
```

Comment:
```suggestion
{% include 'alert/tip', content: 'Admin mutations and queries are used for interacting with instance code. They are an alternative for modifying partials, GraphQL, and others instead of editing files and deploying them with pos-cli. They are useful when you want to build more dynamic behavior.' %}
```

Reply:
Thank you @Slashek!